.changeset/rude-ants-cough.md

```diff
@@ -0,0 +1,9 @@
+---
+"nextjs-website": minor
+"cloudfront-functions": patch
+"cognito-functions": patch
+"eslint-config-custom": patch
+"gitbook-docs": patch
```

Comment:
Is this PR changing all of these packages?

Reply:
Nope i had fix change set